spec/requests/notifications_spec.rb

```diff
@@ -69,19 +71,17 @@
         end
       end
 
-      context 'and the notification to be deleted does not exist' do
+      context 'when notification to be deleted does not exist' do
         let(:notification_id) { 'something-fake' }
 
-        context 'and the requested format is html' do
-          let(:format) { 'html' }
-
+        context 'when requested format is html' do
```

Comment:
you should specify for all html contexts
```ruby
let(:format) { 'text/html' }
```

this ensure the expected behavior is actually happening

Reply:
have made the changes. Thanks for the feedback